portfolio/src/main/webapp/style.css

```diff
@@ -74,7 +84,7 @@ html {
  
 .short {
     margin: auto;
-    width: 450px;
+    width: 250px;
 }
```

Comment:
Do you still use this? I don't see it anywhere.

I do see margin: auto a lot of places though, might consider putting that in its own class so you don't have to add it to individual id styles/etc.